docs/integration.md

```diff
@@ -0,0 +1,248 @@
+
+> TODO: social link handling
+
+## [Sharing](#api-sharing)
+
+Sharing is triggered through `bridget` once again by hijacking the `onclick` event on a specific element which has the specified class.
+
+
+Required class:
+
+|class name|description|optional|
+|---|---|---|
+|webview-social|Indicates that this is a social sharing link and should prompt the native share sheet.|false|
+
+By default, when a social sharing link is clicked, it will try to extract the metadata (url, title, description) from multiple sources in the following order:
+- LDJson metadata
+```html
+<script type="application/ld+json">
+{
+   "@context":"https://schema.org",
+   "@type":"NewsArticle",
+   "mainEntityOfPage":{
+      "@type":"WebPage",
+      "@id":"https://publisher.com/sport/random-article"
+   },
+   "title":"Random article",
+   "datePublished":"2021-07-14T16:55:00+02:00",
+   "dateModified":"2021-07-14T18:03:00+02:00",
+   "description":"Lorem ipsum article...",
+   "author":[
+      {
+         "@type":"Person",
+         "name":"Filip Boev"
+      }
+   ]
+}
+</script>
+```
+- if not present, it will try with Open Graph data
+```html
+    <meta property="og:url" content="https://publisher.com/sport/random-article">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="Random article">
+    <meta property="og:description" content="Lorem ipsum article...">
+```
+- if not present, it will default back to `document`, `location` & plain `meta` tag data.
+```html
+    <meta name="title" content="Random article">
+    <meta name="description" content="SLorem ipsum article...">
+```
+
+### Exposed API
+
+ If you want to manually trigger the native share sheet, then you can use the exposed internal API from `bridget`:
+
+---
+
+### `shareDoc`
+Opens a native share sheet with the supplied spec.
+| Prop  | Required | Type |
+| ------------| -------- | ------------|
+| url         | Yes      | String      |
+| title       | Yes       | String      |
+| description | Yes       | String      |
+```js
+/**
+ * @typedef {object} spec
+ * @property {string} url - the location of the resource
+ * @property {string} title - title of the resource
+ * @property {string} description - description of the resource
+ */
+bridget.shareDoc(spec)
+```
+
+## Hiding specific web components
+
+This part of the API focuses on removing certain components that might appear in the web delivery that don't make sense in a mobile context or have a native counterpart component. 
+
+A few examples that come to mind would be the headers, burger menu, footers, sitemap or anything your web delivery has that **should not appear** in the WebView.
+
+To simply hide an element in the mobile context, you need to add this `css` class to it:
+
+|class name|description|optional|
+|---|---|---|
+|webview-hidden|Whenever this class is added to an html component `bridget`  will inject the CSS property `display: 'none'` to it, thus removing it from the viewport.|false|
```

Comment:
the description about how we do it is not important... just that these elements will be hidden in the context of our app